addons/pinniped/tanzu-auth-controller-manager/Makefile

```diff
@@ -0,0 +1,26 @@
+# This Makefile is here to follow step 4 of this README:
+#   https://github.com/vmware-tanzu/tanzu-framework/blob/5cdba3181afbc31b05d4df51052fb47f3fcb5b92/docs/packages/add-new-package-to-package-repo.md
+
```

Comment:
Should we include the `common.mk` file?

Reply:
can you tell me why you are asking? i see that we can use the `ROOT_DIR` variable, but other than that i don't see anything else we would use from `common.mk` in our Makefile right now

Reply:
If there are not enough useful variables to make it worth including, then I'm ok leaving it out!